app/views/layouts/mailer.html.erb

```diff
@@ -1,21 +1,17 @@
 <html>
+  <head>
+    <%= stylesheet_link_tag 'application/mailer' %>
+  </head>
   <body>
-    <%= yield %>
-
-    <p>
-    -----
-    </p>
-
-    <p>
-      <%= t('layouts.mailer.html.app_description',
-        app_name: link_to(t('app_name'), root_url)
-      ).html_safe %>
-    </p>
-
-    <p>
-      <%= t('layouts.mailer.html.no_reply',
-        email: link_to(t('email'), "mailto:join.ifme@gmail.com")
-      ).html_safe %>
-    </p>
+    <%= render 'shared/mailer_header' %>
+    <div class="grid" id="mailer-container">
+      <div id="mailer-content">
+        <%= yield %>
+        <p>
+        -----
```

Comment:
HmmmMmMmMMmm this is just my opinion but let's leave this out so it's closer to the [quick mock](https://user-images.githubusercontent.com/1121087/48811066-097d8400-ecfa-11e8-9c75-29f0ea38a3bd.png). Open to discussion though!